### DISCUSSION TITLE
Welcome to PL Research Discussions!

### DISCUSSION THREAD

**Discussion by KarolaKirsanow** [protocol/research#34] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

This is the place to discuss:
- ⚗️ MetaResearch and the PL MetaResearch Journal Club 📓 
- ❔ Open Problems that don't fit in the  [CryptoNetLab](https://github.com/protocol/CryptoNetLab) or [ResNetLab](https://github.com/protocol/ResNetLab) repos
- 🎙️ [PL Research Seminars](https://www.youtube.com/watch?v=Dh8_fwk8QBw&list=PLhuBigpl7lqu6xWpiXtbEzJQtlMH1tqoG)

 ✨  Please introduce yourself in the comments! 👋

**Comment by Wondertan** [comment]
It seems like AbstractionLab, CryptoComputeLab, and CryptoEconLab repos aren't public yet to take a closer look. Are they intentionally closed?

**Reply by KarolaKirsanow** [reply]
Those repos are under curently under construction and should be open soon. I'll update the links when they're public. Thanks!
